docs/production.md

```diff
@@ -1,88 +1,267 @@
 
-    ```
-    ...
-    module "cluster" {
-        source = "../../site/tofu/"
+  ```
+  ...
+  module "cluster" {
+      source = "../../site/tofu/"
+      cluster_name = "foo"
+      ...
+  }
+  ```
 
-        cluster_name = "foo"
-        ...
-    }
-    ```
+Note that:
+
+  - Environment-specific variables (`cluster_name`) should be hardcoded into
+    the module block.
+
+  - Environment-independent variables (e.g. maybe `cluster_net` if the same
+    is used for staging and production) should be set as *defaults* in
+    `environments/site/tofu/variables.tf`, and then don't need to be passed
+    in to the module.
+
+## Define and deploy infrastructure
+
+Create an OpenTofu variables file to define the required infrastructure, e.g.:
 
-    Note that:
-        - Environment-specific variables (`cluster_name`) should be hardcoded
-          into the module block.
-        - Environment-independent variables (e.g. maybe `cluster_net` if the
-          same is used for staging and production) should be set as *defaults*
-          in `environments/site/tofu/variables.tf`, and then don't need to
-          be passed in to the module.
+  ```
+  # environments/$ENV/tofu/terraform.tfvars
+  cluster_name = "mycluster"
+  cluster_networks = [
+    {
+      network = "some_network" # *
+      subnet = "some_subnet" # *
+    }
+  ]
+  key_pair = "my_key" # *
+  control_node_flavor = "some_flavor_name"
+  login = {
+      # Arbitrary group name for these login nodes
+      interactive = {
+          nodes: ["login-0"]
+          flavor: "login_flavor_name" # *
+      }
+  }
+  cluster_image_id = "rocky_linux_9_image_uuid"
+  compute = {
+      # Group name used for compute node partition definition
+      general = {
+          nodes: ["compute-0", "compute-1"]
+          flavor: "compute_flavor_name" # *
+      }
+  }
+  ```
+
+Variables marked `*` refer to OpenStack resources which must already exist.
+
+The above is a minimal configuration - for all variables and descriptions see
+`environments/$ENV/tofu/variables.tf`.
+
+To deploy this infrastructure, ensure the venv and the environment are
+[activated](#cookiecutter-instructions) and run:
+
+  ```bash
+  export OS_CLOUD=openstack
+  cd environments/$ENV/tofu/
+  tofu init
+  tofu apply
+  ```
+
+and follow the prompts. Note the OS_CLOUD environment variable assumes that
+OpenStack credentials are defined using a
+[clouds.yaml](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#clouds-yaml)
+file in a default location with the default cloud name of `openstack`.
+
+### Configure appliance
+
+To configure the appliance, ensure the venv and the environment are
+[activated](#create-a-new-environment) and run:
+
+  ```bash
+  ansible-playbook ansible/site.yml
+  ```
+
+Once it completes you can log in to the cluster using:
+
+  ```bash
+  ./dev/ansible-ssh login
+  ```
+
+## Production further configuration
```

Comment:
Open question: does anyone have opinions on ordering here? I'm trying to write the points that roughly relate next to each other, wondering if it could be worth the effort to have a more structured order...